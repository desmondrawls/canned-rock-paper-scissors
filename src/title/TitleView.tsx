import React from 'react';
import { StateProps, DispatchProps } from "./Title";

type TitleProps = StateProps & DispatchProps

export const TitleView: React.SFC<TitleProps> = ({title, onClick}) => {
    return (
        <div onClick={onClick}>{title}</div>
    )
}