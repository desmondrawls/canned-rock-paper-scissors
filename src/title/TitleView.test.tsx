import React from "react";
import {TitleView} from "./TitleView";
import {shallow} from "enzyme";

describe('TitleView', function () {
  it('has not changed during refactoring', function () {
    const subject = shallow(<TitleView {...{title: 'title', onClick: () => {}}}/>)
    expect(subject).toMatchSnapshot();
  });
});