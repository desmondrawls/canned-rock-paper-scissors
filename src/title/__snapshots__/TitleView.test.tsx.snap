// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleView has not changed during refactoring 1`] = `
<div
  onClick={[Function]}
>
  title
</div>
`;
